public/src/main.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { createRoot } from 'react-dom/client';
```

Comment:
https://react.dev/blog/2022/03/08/react-18-upgrade-guide#updates-to-client-rendering-apis